sqlx-core/src/executor.rs

```diff
@@ -120,7 +120,7 @@ pub trait Executor<'c>: Send + Debug + Sized {
             .boxed()
     }
 
-    /// Execute the query and returns at most one row.
+    /// Execute the query and return at most one row.
```

Comment:
I would rephrase this (and other instances of `.fetch_optional()`) as

> Executes the query and returns the first row, if present, or `None` otherwise.
>
> All subsequent rows returned from the query are discarded.